ovos_utils/gui.py

```diff
@@ -475,6 +482,8 @@ def register_handler(self, event, handler):
             event (str):    event to catch
             handler:        function to handle the event
         """
+        if not self.bus:
+            raise RuntimeError("bus not set, did you call self.bind() ?")
```

Comment:
is raising desirable? seems like this could become difficult for an end user to unwind, given that skill logs don't always capture the meaningful part of the traceback

Reply:
the GUI can not work without a bus. This happens only if the object was not fully constructed (bus not passed in init) so this info is targeted as developers not users, it even says what needs to be done :P